.cruft.json

```diff
@@ -23,7 +23,7 @@
             "project_repo_name": "raven",
             "project_readthedocs_name": "pavics-raven",
             "project_short_description": "Raven offers processes related to hydrological modeling, and in particular, the Raven hydrological modeling framework.",
-            "version": "0.12.0",
+            "version": "0.12.1-beta",
```

Comment:
bump again to make it without "beta"?

Reply:
Ah yes, darn. Hate that. I'll wait for the tests to complete. Bump it then merge without waiting for the tests.

Reply:
The docs failed for some reason, but I'll ignore it for now.